pkgs/applications/networking/cluster/argocd/default.nix

```diff
@@ -55,6 +41,12 @@ buildGoModule rec {
     runHook postInstall
   '';
 
+  doInstallCheck = true;
+  installCheckPhase = ''
+    $out/bin/argocd version --client | grep ${tag} > /dev/null
+    $out/bin/argocd-util version --client | grep ${tag} > /dev/null
+  '';
+
   postInstall = ''
     for appname in argocd-util argocd-server argocd-repo-server argocd-application-controller argocd-dex ; do
```

Comment:
```suggestion
```

Because package is restricted to 'cmd'.  Remove:
```
for appname in argocd-util argocd-server argocd-repo-server argocd-application-controller argocd-dex ; do
   makeWrapper $out/bin/argocd $out/bin/$appname --set ARGOCD_BINARY_NAME $appname
done
```